napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -524,18 +768,21 @@ def refresh(self):
         self.available_list.clear()
 
         # fetch installed
-        from npe2 import PluginManager
+        # TODO: Why is this imported here?
+        # from npe2 import PluginManager
 
-        from napari.plugins import plugin_manager
+        # TODO: Why is this imported here?
+        # from napari.plugins import plugin_manager
```

Comment:
This should be solved before merge

Reply:
This was here before, but it doesn't seem necessary.  I don't know who would know the answer to this.  Any thoughts?

Reply:
This TODOs are clearly added in this PR. 

This import looks like it just delayed imports and, for some reason, become obsolete at some moment.

Reply:
oh sorry, I did add the TODO.   It was the imports that were there.  I didn't know why they were needed.